test/testdata/godox.go

```diff
@@ -3,11 +3,11 @@
 package testdata
 
 func todoLeftInCode() {
-	// TODO implement me // ERROR godox.go:6: Line contains FIXME/TODO: "TODO implement me"
```

Comment:
I believe the only thing being matched here was the stuff between double-quotes.